telemetry/request-log/src/main/scala/io/buoyant/telemetry/requestLog/RequestLogAdminHandler.scala

```diff
@@ -0,0 +1,52 @@
+package io.buoyant.telemetry.requestLog
+
+import com.twitter.finagle.http.{MediaType, Request, Response, Status}
+import com.twitter.util.Future
+import io.buoyant.admin.Admin.Handler
+import java.net.URLEncoder
+import java.nio.charset.StandardCharsets.ISO_8859_1
+
+class RequestLogAdminHandler(requestLog: RequestLogTracer) extends Handler {
+
+  override def apply(req: Request): Future[Response] = {
+
+    val tbody = requestLog.getRequests.map { requestMeta =>
+
+      val encodedLogicalName = URLEncoder.encode(requestMeta.logicalName, ISO_8859_1.toString)
```

Comment:
ISO_8859_1 indicates we can only have single byte char in requestMeta, I think it is fine in general.